docs/source/resources.rst

```diff
@@ -30,6 +30,24 @@ Here is how you can register a resource::
             _USERS[len(_USERS) + 1] = self.request.json_body
             return True
 
+Here a example of imperative way defining a resource::
```

Comment:
consider: "Here is an example of how to define cornice resources in an imperative way"